kolibri/core/assets/src/styles/fonts/noto-manifest.json

```diff
@@ -1,34 +1,105 @@
-[
-  {
-    "has_ui_variant": false,
-    "name": "NotoSans"
+{
+  "fonts": {
+    "NotoSans": {
+      "bold_url": "https://raw.githubusercontent.com/googlei18n/noto-fonts/b20c9c07139b22eb38013b1f80f9a5222c6058c6/phaseIII_only/hinted/ttf/NotoSans/NotoSans-Bold.ttf",
```

Comment:
Seems odd that we are storing this info in our assets folder when it is, presumably, only used by the build_tools/i18n functionality?

Reply:
it's not in `/kolibri/core/static`, it's in `/kolibri/core/assets/src/styles/fonts`.

I put it there because I was thinking of it as a "source" file, though in reality it's more similar to stuff in `node_modules` or `site_packages`...

Where do you recommend saving it to?

Reply:
My assumption, because I could only see it being used in build_tools/i18n is that it would be stored in that folder?